sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/Models/LocationListResult.cs

```diff
@@ -12,17 +12,17 @@ internal partial class LocationListResult
         /// <summary> Initializes a new instance of LocationListResult. </summary>
         internal LocationListResult()
         {
-            Value = new ChangeTrackingList<LocationData>();
+            Value = new ChangeTrackingList<Location>();
         }
 
         /// <summary> Initializes a new instance of LocationListResult. </summary>
         /// <param name="value"> An array of locations. </param>
-        internal LocationListResult(IReadOnlyList<LocationData> value)
+        internal LocationListResult(IReadOnlyList<Location> value)
         {
             Value = value;
         }
 
         /// <summary> An array of locations. </summary>
-        public IReadOnlyList<LocationData> Value { get; }
+        public IReadOnlyList<Location> Value { get; }
```

Comment:
Per [https://github.com/Azure/azure-rest-api-specs/blob/7601061bfbb72b2f19cb29c46cbf9a397c2d8893/specification/resources/resource-manager/Microsoft.Resources/stable/2021-01-01/subscriptions.json#L353](https://github.com/Azure/azure-rest-api-specs/blob/7601061bfbb72b2f19cb29c46cbf9a397c2d8893/specification/resources/resource-manager/Microsoft.Resources/stable/2021-01-01/subscriptions.json#L353), shouldn't this be locationExpanded?